data/multiset.lean

```diff
@@ -473,6 +473,9 @@ theorem repeat_subset_singleton : ∀ (a : α) n, repeat a n ⊆ a::0 := repeat_
 theorem repeat_le_coe {a : α} {n} {l : list α} : repeat a n ≤ l ↔ list.repeat a n <+ l :=
 ⟨λ ⟨l', p, s⟩, (perm_repeat.1 p.symm).symm ▸ s, subperm_of_sublist⟩
 
+theorem repeat_one {a : α} : repeat a 1 = a :: 0 :=
+(eq_repeat'.2 (λ b,mem_singleton.1 : ∀ b ∈ a :: 0, b = a)).symm
```

Comment:
This should be `by simp`. Do we not have theorems about `repeat_zero` and `repeat_succ` already?

Reply:
I don't quite understand this comment. If you mean "does

```lean
import data.multiset 

open multiset 

theorem repeat_one {α : Type*} {a : α} : repeat a 1 = a :: 0 := by simp
```

work?" the answer is "not for me". There's `card_repeat` but I can't find `repeat_zero`. I definitely don't know enough about `simp` to make `repeat_one` yield to it. Do you just mean "tag it with simp"?

Reply:
Sorry to be so clueless.

Reply:
I mean that `repeat_zero` and `repeat_succ` should be theorems if they aren't already, from which this follows as a corollary.